src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    protected static final String type = "rectangle";
+    protected double width;
+    protected double height;
+    
+    public Rectangle(String color, double width, double height, String type) {
+        super(color, type);
+        this.width = width;
+        this.height = height;
+    }
+
+    public Rectangle(String color, double width, double height) {
+        this(color, width, height, "rectangle");
+    }
+
+    @Override
+    public double calculateArea() {
+        return width * height;
+    }
+
+    @Override
+    protected String getAdditionalInfo() {
+        return "width: " + width + " units , height: " + height + " units";
+    }
+}
```

Comment:
The class 'Rectangle' should not start with an empty line. Please remove the empty line at the beginning of the class definition.